src/lib/common/limits.h

```diff
@@ -92,20 +96,23 @@
 #define  LOCAL_IP_V4  "0.0.0.0"
 
 
+
 /* ****************************************************************************
 *
 * STATIC_BUFFER_SIZE - to avoid mallocs for "smaller" requests
 */
 #define STATIC_BUFFER_SIZE (32 * 1024) // 32 KB 
 
 
+
 /* ****************************************************************************
 *
 * CONSTANTS RESTINIT - 
 */ 
-#define   DEFAULT_CONNECTION_MEM  	64   
-#define   DEFAULT_MAX_CONNECTIONS   	128   
-#define   DEFAULT_MHD_THREAD_POOLSIZE   128 
+#define   DEFAULT_CONNECTION_MEM  	    64
+#define   DEFAULT_MAX_CONNECTIONS   	  128
+#define   DEFAULT_MHD_THREAD_POOLSIZE   128
+
```

Comment:
I don't know why github presents these lines like this.
Actually I do. There are tabs in the lines ... :-(
Tabs are strongly forbidden. 
One more commit on its way ...